.github/workflows/ci-test-pr.yml

```diff
@@ -0,0 +1,99 @@
+name: Build and Run Integration Test
+
+on:
+  push:
+    branches:
+      - '3875-get-env-var'
```

Comment:
make sure to change this to develop!

Reply:
@dmou, a question: we still want it to run when we are pushing from our local whatever branch, while we are working on it, right? So could it be something like `on: push` or `on: pull_request`?